api/envoy/config/filter/http/rate_limit/v2/rate_limit.proto

```diff
@@ -40,4 +40,8 @@ message RateLimit {
   // communication failure between rate limiting service and the proxy.
   // Defaults to false.
   bool failure_mode_deny = 5;
+
+  // Specifies whether a `RESOURCE_EXHAUSTED` code must be returned instead of
```

Comment:
nit: for explicit clarity I would say `RESOURCE_EXHAUSTED` gRPC code, same with `UNAVAILABLE` and clarify that the http code will still be 429. Also state what the default is.

Reply:
clarification: The HTTP code will be 200 for a gRPC response.